url.bs

```diff
@@ -846,6 +846,14 @@ resource the <a for=/>URL</a>'s other components identify. It is initially null.
 
 <hr>
 
+<p>A <dfn export>non-special-URL host</dfn> is an <a>ASCII string</a> holding data that can be used
+for further processing.
+
+<p class="note no-backref">A <a>non-special-URL host</a> is emphatically not a <a for=/>host</a> and
+only used by <a lt="is special">non-special URLs</a>.
```

Comment:
When you add "not"s it's confusing. "X is not A and B" could be either "X is not A and not B" or "X is not A and is B".